ecmascript/transforms/compat/src/bugfixes/edge_default_param.rs

```diff
@@ -0,0 +1,102 @@
+use swc_ecma_ast::*;
+use swc_ecma_visit::{noop_fold_type, Fold, FoldWith};
+
+/// Converts destructured parameters with default values to non-shorthand
+/// syntax. This fixes the only arguments-related bug in ES Modules-supporting
+/// browsers (Edge 16 & 17). Use this plugin instead of
+/// @babel/plugin-transform-parameters when targeting ES Modules.
+pub fn edge_default_param() -> impl Fold {
+    EdgeDefaultParam::default()
+}
+#[derive(Default, Clone, Copy)]
+struct EdgeDefaultParam {
+    in_arrow: bool,
+}
+
+impl Fold for EdgeDefaultParam {
+    noop_fold_type!();
+
+    fn fold_arrow_expr(&mut self, n: ArrowExpr) -> ArrowExpr {
+        self.in_arrow = true;
+        let params = n.params.fold_with(self);
+        self.in_arrow = false;
+
+        let body = match n.body {
```

Comment:
I think this can be simplified to `n.body.fold_with(self)` or `n.body.fold_children_with(self)`.
Later would have exactly same effect.